packages/types/src/extensions/defaultExtensionConfiguration.ts

```diff
@@ -1,10 +1,9 @@
 import { ChecksumConfiguration } from "./checksum";
+import { RetryStrategyConfiguration } from "./retry";
 
 /**
  * @internal
  *
  * Default extension configuration consisting various configurations for modifying a service client
  */
-export interface DefaultExtensionConfiguration extends ChecksumConfiguration {}
-
-type GetDefaultConfigurationType = (runtimeConfig: any) => DefaultExtensionConfiguration;
```

Comment:
Any risk of `GetDefaultConfigurationType` being removed?

Reply:
No because it's not used